packages/edit-site/src/components/global-styles/font-library-modal/utils/index.js

```diff
@@ -225,31 +230,34 @@ export async function batchInstallFontFaces( fontFamilyId, fontFacesData ) {
 /*
  * Downloads a font face asset from a URL to the client and returns a File object.
  */
-export async function downloadFontFaceAsset( url ) {
-	return fetch( new Request( url ) )
-		.then( ( response ) => {
-			if ( ! response.ok ) {
-				throw new Error(
-					`Error downloading font face asset from ${ url }. Server responded with status: ${ response.status }`
-				);
-			}
-			return response.blob();
-		} )
-		.then( ( blob ) => {
-			const filename = url.split( '/' ).pop();
-			const file = new File( [ blob ], filename, {
-				type: blob.type,
-			} );
-			return file;
+export async function downloadFontFaceAssets( src ) {
+	//src could be a single string or a collection of strings.
+	//here, just make sure it's an array
+	src = Array.isArray( src ) ? src : [ src ];
+
+	const files = await Promise.all(
+		src.map( async ( url ) => {
+			return await fetch( new Request( url ) )
+				.then( ( response ) => {
+					if ( ! response.ok ) {
+						throw new Error(
+							`Error downloading font face asset from ${ url }. Server responded with status: ${ response.status }`
+						);
+					}
+					return response.blob();
+				} )
+				.then( ( blob ) => {
+					const filename = url.split( '/' ).pop();
+					const file = new File( [ blob ], filename, {
+						type: blob.type,
+					} );
+					return file;
+				} );
 		} )
-		.catch( ( error ) => {
-			// eslint-disable-next-line no-console
-			console.error(
-				`Error downloading font face asset from ${ url }:`,
-				error
-			);
-			throw error;
-		} );
```

Comment:
I'm pretty sure we still need this, otherwise `files` may end up begin a rejected promise.

Reply:
The error is handled upstream, this catch simply logs the error to the console (and then re-throws).  The only added benefit of this code is to log the error to the console (which isn't necessary or even desired).  Removing that it's just a catch-and-throw.

Reply:
Ah, nice, I see the upstream error handling in testing this.

Let's put a comment here that error handling is done by `handleInstall`, as a missing catch is potentially a red flag when looking at the code.

Reply:
I couldn't find an example of an async operation not having a catch statement and commenting as such, but quite a few statements that didn't have catch statements.  Please share an example if there's something similar you have in mind.

Reply:
It's come up twice in recent reviews of font library code, so I think we'd save ourselves and others time in the future if a code comment pre-answered the question of how errors are being handled here.